src/cljs/main/broadfcui/page/workspace/create.cljs

```diff
@@ -71,8 +71,9 @@
               "The cloned Workspace will automatically inherit the Authorization Domain from this Workspace."
               [:div {} "You may add Groups to the Authorization Domain, but you may not remove existing ones."]])
            (this :-auth-domain-builder)
-           [comps/ErrorViewer {:error server-error}]
-           (style/create-validation-error-message validation-errors)])}]))
+           [:div {}
```

Comment:
Actually, now that this doesn't have any `flex` or `float` style, do we even need this div?